### PR TITLE
feat(scss) pagination component

### PR DESCRIPTION
**Pull request by BertrandPodevin**
New component for pagination with 2 possibles behaviors : 

Pagination "by clusters" : 
image

Pagination "per page" :
image

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1125

File: packages/scss/demo/components/pagination.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    <nav class="pagination" role="navigation" aria-labelledby="pagination-count">
+        <div id="pagination-count" class="pagination-count">
+            <span class="pagination-count-current">
+                <span class="u-mask">Results from</span>
+                1<span aria-hidden="true">-</span>
+                <span class="u-mask">to</span>10
+            </span>
+            <span class="pagination-count-separator">of</span>
+            <span class="pagination-count-total">
+                50<span class="u-mask">pages</span>
```
Comment (vvalentin-lucca) [inline packages/scss/demo/components/pagination.html:13]: You are missing a space between « 50 » and « pages » here.

File: packages/scss/demo/components/pagination.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
+<h1 class="demo-divider" id="pagination">Pagination</h1>
+<section class="contentSection">
+    <h1>Pagination</h1>
+    <nav class="pagination" role="navigation" aria-labelledby="pagination-count">
+        <div id="pagination-count" class="pagination-count">
+            <span class="pagination-count-current">
+                <span class="u-mask">Results from</span>
+                1<span aria-hidden="true">-</span>
+                <span class="u-mask">to</span>10
```
Comment (vvalentin-lucca) [inline packages/scss/demo/components/pagination.html:9]: You are missing a space between « to» and « 10 » here.

File: packages/scss/demo/components/pagination.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
+<h1 class="demo-divider" id="pagination">Pagination</h1>
+<section class="contentSection">
+    <h1>Pagination</h1>
+    <nav class="pagination" role="navigation" aria-labelledby="pagination-count">
+        <div id="pagination-count" class="pagination-count">
+            <span class="pagination-count-current">
+                <span class="u-mask">Results from</span>
+                1<span aria-hidden="true">-</span>
```
Comment (vvalentin-lucca) [inline packages/scss/demo/components/pagination.html:8]: You are missing a space between « 1» and « - » here. And you should replace your hyphen with a dash.

File: packages/scss/demo/components/pagination.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+                &lt;/button&gt;
+            &lt;/ul&gt;
+        &lt;/nav&gt;
+    </code>
+</section>
+<section>
+    <h2>Per page</h2>
+    <nav class="pagination" role="navigation" aria-labelledby="pagination-navigation">
+        <ul id="pagination-navigation" class="pagination-navigation">
+            <li class="pagination-navigation-item is-active"><span class="u-mask">You're on page</span>1</li>
```
Comment (vvalentin-lucca) [inline packages/scss/demo/components/pagination.html:57]: `aria-current` here maybe?

File: packages/scss/src/components/_pagination.scss [hunk 5]
```diff
@@ -0,0 +1,44 @@
+.pagination {
+    display: flex;
+    flex-wrap: wrap;
+    margin: _theme('spacings.small') 0;
+    > * {
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/_pagination.scss:5]: `:not(:last-child)` ?

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1125

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1125

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1125

**Pull request merged**